kinisi/parser.py

```diff
@@ -85,6 +86,34 @@ def __init__(self,
         self.dimensionality = drift_corrected.sizes['dimension'] * sc.units.dimensionless
 
         self.displacements = drift_corrected['atom', indices]
+    
+    def _to_datagroup(self) -> sc.DataGroup:
+        """
+        Convert the :py:class:`Parser` object to a :py:mod: 'scipp' DataGroup.
+        :return: A :py:mod:`scipp` DataGroup representing the :py:class:`Parser` object.
+        """
+        group = {key: value for key, value in self.__dict__.items()}
```

Comment:
How is this different from?
```suggestion
        group = dict(self.__dict__.items())
```

Reply:
It’s not. Changed now.